package.json

```diff
@@ -34,11 +34,13 @@
     "express-rate-limit": "^5.1.3",
     "express-session": "^1.17.1",
     "extend": "^3.0.2",
+    "geoip-lite": "^1.4.0",
```

Comment:
not necessary, originally was using geoip-lite but switched to web based API from ip-api.com